about.html

```diff
@@ -12,62 +12,87 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+    <div class="container">
+      <div class="container about-page">
+        
+            <section class="top-content">
+            <div>
+                <img class="lambda" src="img\lambda-black.png" alt="lambda-logo">
+            </div>
+            <nav>
+            
+            <a href="index.html">Home</a>
```

Comment:
A few slight formatting issues, using something like Prettier, or another formatter of choice will clean this up.